genesis/process/interface.go

```diff
@@ -0,0 +1,19 @@
+package process
+
+import (
+	"github.com/ElrondNetwork/elrond-go/genesis"
+	"github.com/ElrondNetwork/elrond-go/sharding"
+)
+
+// GenesisParser contains the parsed genesis json file and has some functionality regarding processed data
+type GenesisParser interface {
```

Comment:
the interface should stay under the genesis folder only. as well as werrors.

Reply:
About the interface, there are some cases in which we need to define the interface as closed as possible. Will try to refactor this

Reply:
refactored: moved the interfaces in the genesis package, created a parser package for the genesis parser